draft-ietf-quic-http.md

```diff
@@ -613,17 +613,17 @@ tree could be discarded safely. Clients could potentially reference closed
 streams long after the server had discarded state, leading to disparate views of
 the prioritization the client had attempted to express.
 
-In HTTP/3, a number of placeholders are explicitly permitted by the server using
-the `SETTINGS_NUM_PLACEHOLDERS` setting. Because the server commits to
-maintaining these placeholders in the prioritization tree, clients can use them
-with confidence that the server will not have discarded the state. Clients MUST
-NOT send the `SETTINGS_NUM_PLACEHOLDERS` setting; receipt of this setting by a
-server MUST be treated as a connection error of type
+In HTTP/3, placeholders are given their own number space that spans between zero
+and 2^62-1.  By using the `SETTINGS_NUM_PLACEHOLDERS` setting, the server
+advertises the range of placeholder IDs it is committing to maintain state,
```

Comment:
"advertises the number of placeholder IDs for which it is committing to maintain state."

Reply:
Thank you for the suggestions. I've applied this proposed change (with the omission of "IDs") as well as the one below in 6b8b457, at the same time reordering the sentences so that they flow better.